Wireframe/index.html

```diff
@@ -4,10 +4,50 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+    <header>
+      <h1>Understanding Git</h1>
+      <p>A Brief Overview of Git and Its Key Concepts</p>
+  </header>
+
+  <main>
+      <!-- Full-width blog post box -->
+      <section class="blog-post full-width">
+          <img src="https://images.pexels.com/photos/11035539/pexels-photo-11035539.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2" alt="Git illustration" class="post-image">
+          <div class="content">
+              <h2>What is Git?</h2>
+              <p>Git is a distributed version control system that helps developers track changes in their codebase. It facilitates collaborative development and version management.</p>
+              <a href="#what-is-git" class="read-more">Read More</a>
+          </div>
+      </section>
+
+      <!-- Side-by-side blog post boxes -->
+      <section class="blog-container">
+          <div class="blog-post">
+              <img src="https://images.pexels.com/photos/11035539/pexels-photo-11035539.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2" alt="Git Branch" class="post-image">
+              <div class="content">
+                  <h2>Why Developers Need Git</h2>
+                  <p>Git allows developers to collaborate efficiently, keep track of changes, and resolve conflicts in a team environment. It is essential for large-scale development.</p>
+                  <a href="#why-developers-need-git" class="read-more">Read More</a>
+              </div>
+          </div>
+
+          <div class="blog-post">
+              <img src="https://images.pexels.com/photos/11035539/pexels-photo-11035539.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2" alt="Git Feature Branch" class="post-image">
+              <div class="content">
+                  <h2>What is a Branch in Git?</h2>
+                  <p>A branch in Git represents a separate workspace to develop new features or fix bugs. It keeps changes isolated from the main code until merged.</p>
+                  <a href="#what-is-branch" class="read-more">Read More</a>
+              </div>
+          </div>
+      </section>
+  </main>
+
+  <footer>
```

Comment:
This footer doesn't looked fixed to the bottom of the viewport. Do you need help with this part?

Reply:
@SallyMcGrath Thank you I fixed it ,and now you can check my updated codes.